src/test/java/nl/tudelft/ewi/devhub/webtests/views/AssignmentView.java

```diff
@@ -0,0 +1,126 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+import javax.annotation.Nullable;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Douwe Koopmans on 4-5-16.
+ */
+public class AssignmentView extends ProjectSidebarView {
+
+    private static final By RECENT_ASSIGNMENTS_HEADER = By.xpath("//h4[starts-with(normalize-space(.), 'Assignments')]");
```

Comment:
This header is not present, so the invariant fails. (`Assignment#getName()` is the correct value here). This invalid assertion should be removed.